usaspending_api/broker/management/sql/load_duns.sql

```diff
@@ -30,4 +30,9 @@ CREATE TABLE duns_new AS (
                 ultimate_parent_legal_enti text,
                 duns_id text
             )
-);
\ No newline at end of file
+);
+
```

Comment:
It's good to wrap these 4 lines in a single transaction, so in the event one statement fails no data are deleted
```
BEGIN;
<sql>
<sql>
...
COMMIT;
```